pretyping/retyping.ml

```diff
@@ -301,8 +304,7 @@ let relevance_of_term env sigma c =
       | Fix ((_,i),(lna,_,_)) -> (lna.(i)).binder_relevance
       | CoFix (i,(lna,_,_)) -> (lna.(i)).binder_relevance
       | Proj (p, _) -> Relevanceops.relevance_of_projection env p
-      | Int _ | Float _ -> Sorts.Relevant
-
+      | Int _ | Float _ | Array _ (* FIXME *) -> Sorts.Relevant
```

Comment:
@SkySkimmer same here, I just remove the FIXME?

Reply:
yes